README.md

```diff
@@ -56,11 +56,29 @@ You can watch [the video introduction to Hotwire](https://hotwire.dev/#screencas
 
 ## Compatibility with Rails UJS
 
-Rails UJS includes helpers for sending links and forms over XMLHttpRequest, so you can respond with Ajax. Turbo supersedes this functionality, so you should ensure that you're either running Rails 6.1 with the defaults that turn this off for forms, or that you add `config.action_view.form_with_generates_remote_forms = false` to your `config/application.rb`.
+[Rails UJS](https://github.com/rails/rails/tree/main/actionview/app/assets/javascripts) includes four features that make user interfaces slick. Some work directly with Turbo, while others are superseeded by it.
 
-Note that the helpers that turn `link_to` into remote invocations will _not_ currently work with Turbo. Links that have been made remote will not stick within frames nor will they allow you to respond with turbo stream actions. The recommendation is to replace these links with styled `button_to`, so you'll flow through a regular form, and you'll be better off with a11y compliance.
+#### `data-confirm`
 
-You can still use the `data-confirm` and `data-disable-with`.
+`data-confirm` works in browsers.
+
+If using a native adapter, you may need to implement additional logic, such as a [WKUIDelegate](https://developer.apple.com/documentation/webkit/wkuidelegate#//apple_ref/occ/intfm/WKUIDelegate).
```

Comment:
PR to handle this natively (so no caveat will be needed): https://github.com/hotwired/turbo-ios/pull/19